### ISSUE TITLE
Non existing method

### ISSUE THREAD

**Issue by denisrudov** [supabase-community/supabase-go#12]
# Bug report

## Non existing method

Method doesn't exist. connection to supabase is impossible
<img width="1476" alt="image" src="https://github.com/supabase-community/supabase-go/assets/1747408/351b9a63-0eec-41ef-b7a1-0a140e0cec7b">

in version v0.0.1 all is fine

## To Reproduce

install version v0.02 and try to connect

## Screenshots

<img width="896" alt="image" src="https://github.com/supabase-community/supabase-go/assets/1747408/d158c5b1-b85f-4e1b-ba50-82903e9b24af">

**Comment by ZanzyTHEbar** [comment]
Yup, same here. Seems that the proper method is `TokenAuth` not `SetAuthToken`

**Comment by ZanzyTHEbar** [comment]
Strangely enough, when i use the proper method (as found in the postgres-go client.go file) i get: 
![image](https://github.com/supabase-community/supabase-go/assets/45744329/29710d14-3dbe-484d-b4ea-c2e08005b2c0)
Which is a bit confusing ...

**Comment by ZanzyTHEbar** [comment]
I "fixed" the issue my manually cloning [supabase-community/postgrest-go](https://github.com/supabase-community/postgrest-go) into my go package cache.

**Comment by muratmirgun** [comment]
Hello, I will check it. I will write updates here!

**Comment by tranhoangvuit** [comment]
Hi @denisrudov, @ZanzyTHEbar sorry for the missing information. We separated `TokenAuth` into `SetApiKey` and `SetAuthToken` for better readability in the new postgers-go tag [v0.0.11](https://github.com/supabase-community/postgrest-go/releases/tag/v0.0.11)
Can you use latest version(v0.0.4) with new function?
